e2e/testsuite/grpc_query.go

```diff
@@ -49,14 +49,14 @@ func (s *E2ETestSuite) QueryPacketCommitment(ctx context.Context, chain ibc.Chai
 // QueryInterchainAccount queries the interchain account for the given owner and connectionId.
 func (s *E2ETestSuite) QueryInterchainAccount(ctx context.Context, chain ibc.Chain, owner, connectionId string) (string, error) {
 	queryClient := s.GetChainGRCPClients(chain).ICAQueryClient
-	res, err := queryClient.InterchainAccount(ctx, &intertxtypes.QueryInterchainAccountRequest{
+	res, err := queryClient.InterchainAccount(ctx, &controllertypes.QueryInterchainAccountRequest{
```

Comment:
will this break cross compatibility tests?

Reply:
Yeah, great catch! 

I've re-added the intertx query client and a `QueryInterchainAccountLegacy` method which is used by the existing tests